cores/nRF5/avr/pgmspace.h

```diff
@@ -102,7 +102,7 @@ typedef const void* uint_farptr_t;
 #define pgm_read_word(addr) (*(const unsigned short *)(addr))
 #define pgm_read_dword(addr) (*(const unsigned long *)(addr))
 #define pgm_read_float(addr) (*(const float *)(addr))
-#define pgm_read_ptr(addr) (*(const void *)(addr))
+#define pgm_read_ptr(addr) (*(const void* *)(addr))
```

Comment:
Yeah, it is spot on with samdcore https://github.com/arduino/ArduinoCore-samd/blob/master/cores/arduino/avr/pgmspace.h#L106

You should also file an issue on this original repo as well to help them out https://github.com/sandeepmistry/arduino-nRF5/blob/master/cores/nRF5/avr/pgmspace.h#L105

Reply:
Agree, I've made https://github.com/sandeepmistry/arduino-nRF5/pull/418